lib/hydra/derivatives/services/capability_service.rb

```diff
@@ -3,12 +3,21 @@
 module Hydra::Derivatives
   class CapabilityService
     attr_accessor :ffmpeg_output
-    def initialize
+    def capture_output
       @ffmpeg_output = Open3.capture3('ffmpeg -codecs').to_s
     end
 
     def fdk_aac?
       @ffmpeg_output.include?('--enable-libfdk-aac') || @ffmpeg_output.include?('--with-fdk-aac')
     end
+
+    def settings
```

Comment:
I think I prefer this back in video processor config since this capability service could be of value outside of ffmpeg.